R/fit_model.R

```diff
@@ -63,7 +63,7 @@ fit_model <- function(
         rt = rt,
         gp = gp,
         stan = stan,
-        verbose = TRUE,
+        verbose = FALSE,
```

Comment:
Does this lose other types of logs from `EpiNow2`? I think we want it to be on the more verbose side of things

Reply:
It's still[ logging INFO and above](https://github.com/CDCgov/cfa-epinow2-pipeline/actions/runs/12381545455/job/34560332827#step:6:81). It's only dropping DEBUG.